src/PKSim.Core/Model/ParameterExtensions.cs

```diff
@@ -216,4 +237,26 @@ public static void ScaleDistributionBasedOn(this IDistributedParameter parameter
          parameter.IsFixedValue = false;
       }
    }
+
+   public static class ExpressionParameterExtensions
+   {
+      public static string ContainerNameForRelativeExpressionParameter(this ExpressionParameterUpdate expressionParameter)
+      {
+         if (expressionParameter.IsGlobalExpression())
+            return CoreConstants.ContainerName.GlobalExpressionContainerNameFor(expressionParameter.Name);
+         
+         var objectPath = expressionParameter.Path;
+         var containerName = parentOfParent(objectPath);
+         var key = Equals(containerName, CoreConstants.Compartment.INTRACELLULAR) ? objectPath[objectPath.Count - 4] : containerName;
```

Comment:
Is this code that is/was in PKSim already?

Reply:
Not exactly, It follows the code from

```AllExpressionParametersFor``` from ```Individual.cs```
But this is using object paths instead of IParameters in containers.

Reply:
I don't like this 4 without any comment ad to why this is like this. If it was like this already, it's poor from (me probably)